### DISCUSSION TITLE
Lab5 執行 source run_vivado_gcd.sh 出現錯誤

### DISCUSSION THREAD

**Discussion by sssh311318** [bol-edu/HLS-SOC-Discussions#147] [Q&A]
老師好，助教好，
如題，
大致上看了warming 好像是interface有出現問題，
如下圖：
![vivado_error_1](https://github.com/bol-edu/HLS-SOC-Discussions/assets/59499001/3fe044c7-c47f-4458-bb68-f8470e7d6c99)
![vivado_error_2](https://github.com/bol-edu/HLS-SOC-Discussions/assets/59499001/59c97172-ba54-4b9e-9a10-af467f1c9cb6)

但是執行source run_vivado.sh是沒問題的，
產出的檔案也可以在jupyter notebook上跑，
所以不太確定問題出在哪邊，
附檔為vivado.log file，
謝謝。
[vivado.log](https://github.com/bol-edu/HLS-SOC-Discussions/files/13414932/vivado.log)

**Comment by DanielDu9453** [comment]
我這邊是把implement的job數改小後可以成功運行
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/113331381/da899deb-bdb4-4c46-9405-fbea8192874f)
你可以嘗試看看

**Reply by sssh311318** [reply]
謝謝，有成功輸出bitstream了。
